core/services/vrf/v2/listener_v2.go

```diff
@@ -503,8 +503,14 @@ func (lsn *listenerV2) processPendingVRFRequests(ctx context.Context) {
 		// first. This allows us to break out of the processing loop as early as possible
 		// in the event that a subscription is too underfunded to have it's
 		// requests processed.
-		slices.SortFunc(reqs, func(a, b pendingRequest) bool {
-			return a.req.CallbackGasLimit() < b.req.CallbackGasLimit()
+		slices.SortFunc(reqs, func(a, b pendingRequest) int {
+			if a.req.CallbackGasLimit() == b.req.CallbackGasLimit() {
+				return 0
+			}
+			if a.req.CallbackGasLimit() < b.req.CallbackGasLimit() {
+				return -1
+			}
+			return 1 // a.req.CallbackGasLimit() > b.req.CallbackGasLimit()
```

Comment:
Could this be reduced to a subtraction?

Reply:
I don't think so, these are unsigned integers so there will be overflow.

Reply:
Oh, what about:
```suggestion
			return cmp.Compare(a.req.CallbackGasLimit(), b.req.CallbackGasLimit())
```

Reply:
Yup that looks cleaner 👍